src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
```

Comment:
When specifying a font family, it's a good practice to include a fallback font in case the main one doesn't load correctly. In this case, `Avenir` is not a standard font, so you should include a fallback like `sans-serif`. (See the CSS section of the checklist)